arch/arm64/boot/dts/qcom/msm8953-xiaomi-ysl.dts

```diff
@@ -0,0 +1,124 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/dts-v1/;
+
+#include "msm8953-xiaomi-common.dtsi"
+
+/delete-node/ &adsp_fw_mem;
+/delete-node/ &cont_splash_mem;
+/delete-node/ &qseecom_mem;
+/delete-node/ &wcnss_fw_mem;
+
+/ {
+	model = "Xiaomi Redmi S2/Y2 (ysl)";
```

Comment:
remove `(ysl)` from this string

And do I understand it correctly, the Redmi S2 https://www.mi.com/uk/redmi-s2/ and the Redmi Y2 https://www.mi.com/in/redmi-y2/ is really just the same device (for different markets or so?)

Reply:
Yes as far as know there is no difference between S2 and Y2. But i can remove the Y2 from the model string.

Reply:
@z3ntu Which one is better keep "Xiaomi Redmi S2/Y2" or "Xiaomi Redmi S2"?

Reply:
Y2 has same codename ysl

Reply:
I think it's fine